CMakeLists.txt

```diff
@@ -296,3 +296,7 @@ set(CPACK_PACKAGE_FILE_NAME "scap-security-guide-${SSG_VERSION}")
 
 set(CPACK_GENERATOR "RPM;DEB")
 include(CPack)
+
+enable_testing()
+FILE(GLOB_RECURSE datastreams "ssg-*-ds.xml")
```

Comment:
Do we need to use `GLOB_RECURSE`? The output datastreams are always in `build` directory.

Reply:
Interestingly enough, when I replace `GLOB_RECURSE` with `GLOB`, no files are found and I had to replace it back in order to get it working.

Reply:
Maybe you need to scan `${CMAKE_BINARY_DIR}`.

Reply:
you are likely not using the right path, a good practice in cmake is to avoid implicit paths and always list absolute explicit paths. it's easier to read and more reliable when moving code around